client/src/redux/selectors.js

```diff
@@ -57,8 +57,7 @@ export const shouldRequestDonationSelector = state => {
   if (
     showMultipleProgressModals &&
     progressDonationModalShown &&
-    completedChallengesLength > 50 &&
-    completionCount - lastCompletionCount >= 30
+    completionCount - lastCompletionCount >= 20
```

Comment:
Nit: The comment above this needs to be updated to match (_cannot make suggestion_ 😞 )